packages/@coorpacademy-components/src/util/check-is-mobile.js

```diff
@@ -0,0 +1,3 @@
+const mobileUserAgentRegex = /iphone|ipad|ipod|android|blackberry|windows phone/g;
+
+export const isMobile = () => mobileUserAgentRegex.test(navigator.userAgent.toLowerCase());
```

Comment:
quality, test it!
(`mobileUserAgentRegex` to `CONSTANT_CASE`

```suggestion
export const isMobile = (userAgent = navigator.userAgent) => mobileUserAgentRegex.test(userAgent.toLowerCase());
```